nodes/src/nodes/audio_tts/tts_engine.py

```
@@ -0,0 +1,547 @@
+                        {
+                            'text': text,
+                            'voice': str(self.config.get('voice', '') or '').strip(),
+                            'model': str(self.config.get('model', 'eleven_multilingual_v2') or '').strip(),
+                        }
+                    ],
+                    'output_fields': ['audio_base64', 'mime_type'],
+                },
+            )
+            rows = body.get('result') or []
+            if not rows:
+                raise ValueError('ElevenLabs TTS model server returned no result')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)